resources/views/layouts/default/flights/table.blade.php

```diff
@@ -1,91 +1,113 @@
+				@if($flight->arr_time), {{ $flight->arr_time }}@endif
+				<br/>
+				@if($flight->distance)
+					<span class="title">{{ strtoupper(__('common.distance')) }}&nbsp;</span>
+					{{ $flight->distance }} {{ setting('units.distance') }}
+				@endif
+				<br/>
+				@if($flight->level)
+					<span class="title">{{ strtoupper(__('flights.level')) }}&nbsp;</span>
+					{{ $flight->level }} {{ setting('units.altitude') }}
+				@endif
+			</div>
+			<div class="col-sm-5">
+			@if($flight->route)
+				<span class="title">{{ strtoupper(__('flights.route')) }}&nbsp;</span>
+            	{{ $flight->route }}
+			@endif
+			</div>
+		</div>
+		<div class="row">
+			<div class="col-sm-12 text-center">
+				<br>
+				<div class="row">	
+				@if ($simbrief !== false)
+					@if ($simbrief_bids === false || ($simbrief_bids === true && in_array($flight->id, $saved, true)))
+					<div class="col-sm-6 text-left" style="vertical-align: bottom">
+					<select id="AircraftSelection" class="form-control select2" onchange="OnChangeACSelection()">
+						<option value="ZZZZZ">Please Select An Aircraft</option>
+						@if($flight->subfleets)
+							@foreach($flight->subfleets as $SubFleets)
+								@foreach($SubFleets->aircraft as $AC)
+									<option value="{{ $AC->id }}">[ {{ $AC->icao }} ] {{ $AC->registration }} @if($AC->registration <> $AC->name) '{{ $AC->name }}' @endif</option>
+								@endforeach
+							@endforeach
+						@endif
+					</select>
+					</div>
+					<div class="col-sm-3" style="vertical-align: middle">		
+						<a id="SbFormLink" style="visibility: hidden" href="{{ route('frontend.simbrief.generate') }}?flight_id={{ $flight->id }}" class="btn btn-sm btn-primary">Proceed To Flight Planning</a>
+					</div>
+					<div class="col-sm-3" style="vertical-align: middle">
+						<a href="{{ route('frontend.pireps.create') }}?flight_id={{ $flight->id }}" class="btn btn-sm btn-info">{{ __('pireps.newpirep') }}</a>
+					</div>
+					@endif
+				@else
+					<hr>
+					<div class="col-sm-6" style="vertical-align: baseline">&nbsp;</div>
+					<div class="col-sm-3" style="vertical-align: baseline">&nbsp;</div>
+					<div class="col-sm-3" style="vertical-align: baseline">
+						<a href="{{ route('frontend.pireps.create') }}?flight_id={{ $flight->id }}" class="btn btn-sm btn-info">{{ __('pireps.newpirep') }}</a>
+					</div>
+				@endif
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+<script type="text/javascript">
+		// *** Simple Aircraft Selection With Dropdown Change
+		// *** Also keep Generate button hidden until a valid AC selection
+		function OnChangeACSelection() {
+			if (document.getElementById("AircraftSelection").value == "ZZZZZ") {
+					document.getElementById('SbFormLink').style.visibility = 'hidden';
+				}else{
+					document.getElementById('SbFormLink').style.visibility = 'visible';
+				}
+			var $OldLink = document.getElementById("SbFormLink").href ;
```

Comment:
Use `const` instead of `var`

Reply:
fixed the javascript logic and variable names in table.blade.php , sent it as a commit to this pr.